core/scss/components/breadcrumbs/breadcrumbs.scss

```diff
@@ -11,7 +11,7 @@
 
     li {
       font-family: $brand-font-regular;
-
+      cursor: pointer;
```

Comment:
It should be default cursor for the last item(not-clickable).

Reply:
the current class gets applied to the last-item which applies the default cursor @bfbiggs 

.md-breadcrumbs>.current {
    color: #24282b;
    cursor: default;
}